src/components/PlayerControls.tsx

```diff
@@ -29,7 +29,7 @@ const PlayerControls: FunctionComponent<PlayerControlsProps> = ({
   isPro,
 }: PlayerControlsProps) => {
   console.log('nextLessonUrl', nextLessonUrl)
-  const [speed, setSpeed] = useState(1)
+  const [speed, setSpeed] = useState<string>('1')
```

Comment:
the React hooks use some format of generic types and this has been my approach. If you use a custom style it would go here.

Reply:
https://fettblog.eu/typescript-react/hooks/ this is a great bookmark